torch/csrc/jit/script/compiler.cpp

```diff
@@ -1236,21 +1236,20 @@ struct to_ir {
   }
 
   // *********************** Loop Operators ************************************
-  // Emits a loop operators conforming to the semantics specified at
+  // Emits a loop operator with the form:
+  // Loop(max_trip_count)
+  // block0(loop_counter) {
+  //   <body>
+  // }
+  // block1 {
```

Comment:
I worry about putting this block second, because in all other cases sub-blocks are written in lexical order. This may lead to some weird corner cases when naming items in the loop, creating lists of uses, etc.